### PR TITLE
[11.0] [MIG] sale_quotation_number

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Migration of module Sale Quotation Number to v11

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
Please squash OCA Transbot commits into 1, and yours into 1

File: sale_quotation_number/data/data.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright 2010-2012 Andy Lu <andy.lu@elico-corp.com> (Elico Corp)
+Copyright 2013 Agile Business Group sagl (<http://www.agilebg.com>)
+Copyright 2017 valentin vinagre  <valentin.vinagre@qubiq.es> (QubiQ)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+-->
+
+<odoo>
+    <data noupdate="1">
```
Comment (nikul-serpentcs) [inline sale_quotation_number/data/data.xml]: @murtuzasaleh IMHO direct use` <odoo noupdate="1">`

**Comment by tafaRU** [comment]
@murtuzasaleh please resolve conflicts.

File: sale_quotation_number/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# Copyright 2010-2012 Andy Lu <andy.lu@elico-corp.com> (Elico Corp)
```
Comment (nikul-serpentcs) [inline sale_quotation_number/models/__init__.py]: @murtuzasaleh IMHO remove copyright from all  __init__.py

**Review by nikul-serpentcs — APPROVED** [review]
Code review LGTM

**Review by JayVora-SerpentCS — APPROVED** [review]

**Pull request merged**